### DISCUSSION TITLE
tailwind not working

### DISCUSSION THREAD

**Discussion by sysmat** [web-infra-dev/rsbuild#2758] [Q&A]
- I migrate from vite
- I follow : https://rsbuild.dev/guide/basic/tailwindcss
- in build I don't see tailwind css 
- in dev tailwind is not applayed
- postcss.config.cjs:
```
module.exports = {
  plugins: {
    tailwindcss: {},
  },
};
```
-  tailwind.config.js

```
/** @type {import('tailwindcss').Config} */
module.exports = {
  content: ['./src/**/*.{html,js,ts,jsx,tsx}'],
  theme: {
    extend: {},
  },
  plugins: [],
};
```
- main.css
```
@tailwind base;
@tailwind components;
@tailwind utilities;
```

PS: everyting workss fine with **vite**

**Comment by chenjiahan** [comment]
Hi, can you produce a reproduction?

Or you can refer to this example to find the difference: https://github.com/rspack-contrib/rspack-examples/tree/main/rsbuild/react-tailwindcss

**Comment by sysmat** [comment]
@chenjiahan thx, I was missing import main.css in main.ts

**Reply by sysmat** [reply]
with **"@rsbuild/core": "^1.0.0-alpha.5"** it is not working, with **"@rsbuild/core": "0.7.10"** it is working, is any breaking changes?

**Reply by chenjiahan** [reply]
see: https://github.com/web-infra-dev/rsbuild/discussions/2837

**Reply by sysmat** [reply]
1.0.0-alpha.5 not working

**Reply by sysmat** [reply]
aha, now I see, lock version

```json
"@rsbuild/core": "1.0.0-alpha.5",
"@rsdoctor/rspack-plugin": "^0.3.7",
```
